content/applications/sales/point_of_sale/payment_methods/terminals/viva_wallet.rst

```diff
@@ -0,0 +1,139 @@
+
+.. image:: viva_wallet/api-cred.png
+   :alt: Client secret and client ID fields
+
+.. note::
+   These credentials are used for Android and iOS POS Activation requests, as well as the Cloud
+   Terminal API.
+
+.. _viva_wallet/identifier:
+
+Terminal ID
+~~~~~~~~~~~
+
+Your terminal ID number is used to identify your terminal. To find it:
+
+#. Go to your Viva Wallet account and select the relevant account.
+#. Go to :menuselection:`Sales --> Physical payments --> Card terminals` in the navigation menu.
+
+The terminal ID number is located under the :guilabel:`Terminal ID (TID)` column. Save it to paste
+it into the :guilabel:`Terminal ID` field :ref:`when creating the payment method
+<viva_wallet/method-creation>`.
+
+.. image:: viva_wallet/terminal-id.png
+   :alt: Viva terminal ID
+
+.. _viva_wallet/method-creation:
+
+Configure the payment method
+----------------------------
+
+Enable the payment terminal in the :guilabel:`Payment Terminals` section :ref:`of the application
+settings <configuration/settings>`.
+
+Then, go to :menuselection:`Point of Sale --> Configuration --> Payment Methods`, and click
+:guilabel:`New` to :doc:`create the related payment method <../../payment_methods>`. Set the journal
+type as :guilabel:`Bank` and select :guilabel:`Viva Wallet` in the :guilabel:`Use a Payment
+Terminal` field.
+
+Finally, fill in the mandatory fields with your
+
+- :ref:`Merchant ID and API key <viva_wallet/ID-key>`
+- :ref:`Client ID and Client secret <viva_wallet/pos-apis>`
+- :ref:`Terminal ID <viva_wallet/identifier>`
+
+.. image:: viva_wallet/create-method-viva-wallet.png
+   :alt: payment method creation form
+   :scale: 75%
+
+Then, save the form and copy the generated webhook URL of the :guilabel:`Viva Wallet Webhook
+Endpoint` field. This URL is necessary :ref:`when configuring the Webhook <viva_wallet/webhook>`.
+
+.. note::
+   Should the Viva Wallet payment method be missing in the application settings,
+   :doc:`activate the POS Viva Wallet module <../../../../general/apps_modules>` to enable it.
+
+.. _viva_wallet/webhook:
+
+Configure the Webhook
+---------------------
+
+Webhooks allow you to receive real-time notifications whenever a transaction occurs within your Viva
+Wallet account. Set them up for `payment transactions following the Viva documentation
+<https://developer.vivawallet.com/webhooks-for-payments/transaction-payment-created/>`_.
+
+.. seealso::
+   - `Setting up webhooks <https://developer.viva.com/webhooks-for-payments/#setting-up-webhooks>`_
+
+Link the payment method to a POS
+--------------------------------
+
+You can select the payment method in your POS settings once the payment method is created. To do so,
+go to the :ref:`POS' settings <configuration/settings>`, click :guilabel:`Edit`, and add the payment
+method under the :guilabel:`Payment methods` field of the :guilabel:`Payments` section.
+
+Pay with a payment terminal
```

Comment:
Is this process actually the same for all payment terminals? If that's the case, maybe it could be moved to the main Payment terminals page to avoid repetitions? Just a sugegstion, and absolutely no need to do it in this PR of course.